neptune/internal/streams/channel_writer.py

```diff
@@ -53,3 +53,26 @@ def write(self, data):
             )
 
         self._data = lines[-1]
+
+
+class TimeOffset(object):
```

Comment:
Maybe TimeOffsetGenerator name would be better? would explain the .next() call

Reply:
Done